packages/xo-web/src/common/xo/index.js

```diff
@@ -3796,3 +3798,29 @@ const _callGithubApi = async (endpoint = '') => {
 export const getMasterCommit = () => _callGithubApi('/commits/master')
 
 export const compareCommits = (base, head) => _callGithubApi(`/compare/${base}...${head}`)
+
+export const uuidToLink = text => {
+  const regex = /\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b/g
+  const parts = text.split(regex)
+  const uuids = text.match(regex) || []
+  const state = store.getState()
+
+  return flatMap(parts, (part, index) => {
+    const uuid = uuids[index]
+    if (uuid) {
```

Comment:
Explicit condition.
```suggestion
    if (uuid !== undefined) {
```

Reply:
Is this check necessary?

Reply:
@pdonias It's necessary to avoid adding a `<Copiable />` component by default (l. 3820) and have a duplicate component on the whole log message. I've tried other solutions, but the behavior for copying the message was not the same.